modules/ROOT/pages/_partials/common-read-operation.adoc

```diff
@@ -1,9 +1,9 @@
-== Mime Types and Encoding
+== MIME Types and Encoding
 
-This connector tries to determine a file’s mime type (`mimeType`) based on its extension. However, when first-hand knowledge of the file’s content is necessary, you can force that `mimeType` to a particular value by using the `outputMimeType` parameter.
+Each connector tries to determine a file’s MIME type (`mimeType`) from its extension. However, if first-hand knowledge of the file’s content is necessary, you can force the MIME type to a different value with the `outputMimeType` parameter.
 
-The same process works for encoding. By default, the connector assumes that Mule runtime’s default encoding matches the one in the file. However, you can set the encoding through the `outputEncoding` parameter.
+The same process works for encoding. By default, the connector assumes that the Mule runtime’s default encoding matches the MIME type of the file. However, you can set the encoding through the `outputEncoding` parameter.
 
-== Why Mime Types Matter
+== Why MIME Types Matter
 
-DataWeave expressions are embeddable inside operations that generate payloads and other values. Having the correct `mimeType` set helps DataWeave auto-assign types and, in turn, also generate the correct outputs. It also improving the user experience by maximizing the use of DataSense’s functionality.
\ No newline at end of file
+DataWeave expressions are embeddable inside operations that generate payloads and other values. Having the correct `mimeType` set helps DataWeave auto-assign types and also generate the correct outputs. Also maximizing the use of DataSense’s functionality improves the user experience.
```

Comment:
"Also maximizing" -> "Also, maximizing"

Reply:
Changed to:
Also, maximizing the use of DataSense’s functionality improves the user experience.